usage/tests/org.jboss.tools.usage.test/META-INF/MANIFEST.MF

```diff
@@ -10,7 +10,8 @@ Require-Bundle: org.jboss.tools.usage;bundle-version="1.0.0",
  org.eclipse.osgi;bundle-version="3.5.2",
  org.eclipse.core.runtime;bundle-version="3.5.0",
  org.junit;bundle-version="[4.8.1,5.0.0)",
- org.eclipse.jface;bundle-version="3.10.0"
+ org.eclipse.jface;bundle-version="3.10.0",
+ org.hamcrest.core;bundle-version="1.3.0"

```

Comment:
This bundle makes explicit use of hamcrest APIs. The version of junit in my environment does not re-export hamcrest, so an explicit requirement is needed. This has the added bonus of protecting you from build failures should the version of junit in orbit ever change to be more in line with upstream junit.